src/sentry/api/serializers/models/project.py

```diff
@@ -684,8 +684,10 @@ def get_attrs(
             attrs[item]["has_user_reports"] = item.id in projects_with_user_reports
             if not self._collapse(LATEST_DEPLOYS_KEY):
                 attrs[item]["deploys"] = deploys_by_project.get(item.id)
-            attrs[item]["symbolication_degraded"] = realtime_metrics.is_lpq_project(
-                project_id=item.id
+            # check if the project is in LPQ for any platform
+            attrs[item]["symbolication_degraded"] = any(
+                should_demote_symbolication(platform, project_id=item.id)
+                for platform in ["native", "js", "jvm"]
```

Comment:
I advocated for just removing this flag (initially setting to `false` for backwards compat), so might be the time to consider that?